pkg/gomod/file/parser_test.go

```diff
@@ -0,0 +1,32 @@
+package file
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestFileParser_ModuleName(t *testing.T) {
+	expectedModuleName := "arschles.com/testmodule"
+
+	parser := NewFileParser("../../../testmodule/go.mod")
+
+	got, err := parser.ModuleName()
+	if err != nil {
+		t.Fatalf("Expected to find a module name... Got %v", err)
+	}
+
+	if !strings.EqualFold(got, expectedModuleName) {
+		t.Fatalf(`Module names do not match.. \n 
+Expected %s .. Got %s`, expectedModuleName, got)
+	}
+}
+
+func TestFileParser_FileNotFound(t *testing.T) {
+	parser := NewFileParser("../../../go.mod")
+
+	_, err := parser.ModuleName()
+	if !os.IsNotExist(err) {
+		t.Fatalf("file does not exist but got %v instead", err)
+	}
+}
```

Comment:
nit/note for next time: let's write tests with [assert](https://godoc.org/github.com/stretchr/testify/assert)/[require](https://godoc.org/github.com/stretchr/testify/require) helpers and, as packages start to get bigger, use [suite](https://godoc.org/github.com/stretchr/testify/suite).

Overall, these packages shorten test code and make it more readable. Not blocking for now

Reply:
Got that